pkgs/applications/science/misc/sumo/default.nix

```diff
@@ -0,0 +1,56 @@
+{ bzip2, cmake, eigen, fetchFromGitHub, ffmpeg, fox_1_6, gdal, git, gl2ps, gpp
+, gtest, jdk, libGL, libGLU, libX11, libjpeg, libpng, libtiff, openscenegraph
+, proj, python3, python37Packages, stdenv, swig, xercesc, xorg, zlib }:
+
+with stdenv.lib;
+
+stdenv.mkDerivation rec {
+
+  pname = "sumo";
+  version = "1.8.0";
+
+  src = fetchFromGitHub {
+    owner = "eclipse";
+    repo = "sumo";
+    rev = "4820708f3db155db9001523cac05803ceadceeff";
```

Comment:
why not using version, they dont tag releases ?

Reply:
I usually do not track releases, but it makes more sense for this PR. Fixed.